utils/print_config.go

```diff
@@ -0,0 +1,10 @@
+package utils
```

Comment:
I'd probably move `serve.go` to this one, and just make it `config.go` where configurations can be set/printed. What do you think?